quantities/build.gradle.kts

```diff
@@ -0,0 +1,12 @@
+plugins {
+    id("com.android.library")
+    id("kotlin-android")
+    id("VitaminComposeLibraryPlugin")
+    id("com.vanniktech.maven.publish")
+}
+
+dependencies {
+    api(project(":foundation"))
+    api(project(":text-inputs"))
```

Comment:
I think you though you'll need this dependency to develop the quantity component but you have used `BasicTextField` (from my opinion, the best choice!). You can remove this line.